doc/whats_new/v0.20.rst

```diff
@@ -763,7 +763,7 @@ Support for Python 3.3 has been officially dropped.
   implements the Yeo-Johnson and Box-Cox power transformations. Power
   transformations try to find a set of feature-wise parametric transformations
   to approximately map data to a Gaussian distribution centered at zero and
-  with unit variance.  This is useful as a variance-stabilizing transformation
+  with unit variance. This is useful as a variance-stabilizing transformation
   in situations where normality and homoscedasticity are desirable.
   :issue:`10210` by :user:`Eric Chang <ericchang00>` and :user:`Maniteja
```

Comment:
The username is `chang` (cf https://github.com/scikit-learn/scikit-learn/pull/11520)

Reply:
@rth Resolved in https://github.com/scikit-learn/scikit-learn/commit/4fa7c3850d251b3f17112147ede25d07d92df2f5 :)